independent-projects/tools/common/src/test/resources/templates/dockerfile-jvm.ftl

```diff
@@ -40,4 +40,4 @@ COPY ${build_dir}/*-runner.jar /deployments/app.jar
 EXPOSE 8080
 USER 1001
 
-ENTRYPOINT [ "/deployments/run-java.sh" ]
\ No newline at end of file
+ENTRYPOINT [ "/deployments/run-java.sh" ]
```

Comment:
This line change doesn't seem to change anything, can you revert it please?

Reply:
is it better now ?

Reply:
The change still shows up in the GitHub UI.

Moreover, once the proper revert is in place, we'll need the commits to be squashed

Reply:
I did the modifs from github UI.  I'll just do it from Intellij.